tests/unit/dom/fill-in-test.js

```diff
@@ -44,6 +44,26 @@ module('DOM Helper: fillIn', function(hooks) {
     );
   });
 
+  test('filling in a disabled element', async function(assert) {
+    element = buildInstrumentedElement('input');
+    element.dataset.testDisabled = '';
+    element.setAttribute('disabled', true);
+
+    await setupContext(context);
+
+    assert.rejects(
+      fillIn(`[data-test-disabled]`, 'foo'),
+      new Error("Can not `fillIn` disabled '[data-test-disabled]'."),
+      'renders user selector'
+    );
+
+    assert.rejects(
+      fillIn(element, 'foo'),
+      new Error("Can not `fillIn` disabled '[object HTMLInputElement]'."),
```

Comment:
@Turbo87 just improved error messages. See this one, and 1 assert above. Please let me know if it looks good for you.